docs/GettingStarted.md

```diff
@@ -37,14 +37,17 @@ install it directly from [here](https://www.microsoft.com/en-us/download/details
  "*Windows Driver Kit Visual Studio extension*" (make sure the "*Install Windows Driver Kit Visual Studio Extension*"
   check box is checked before completing the installer).
     >Note: as multiple versions of WDKs cannot coexist side-by-side, you may be asked to uninstall previous versions.
-1. [Clang for Windows 64-bit](https://github.com/llvm/llvm-project/releases/download/llvmorg-11.0.1/LLVM-11.0.1-win64.exe) (version **11.0.1**).
- Note: clang versions 12 and higher are NOT yet supported, as they perform program optimizations that are incompatible with the PREVAIL verifier.
 1. [NuGet Windows x86 Commandline](https://www.nuget.org/downloads) (version **6.3.1 or higher**), which can be installed to a location
  such as "C:\Program Files (x86)\NuGet\".
 
 You should add the paths to `git.exe`, `cmake.exe` and `nuget.exe` to the Windows PATH environment variable after the software packages
  above have been installed.
 
+The following are optional components that you can install for additional functionality:
```

Comment:
PR #2673 only affected the cmake build.  What about the Visual Studio build?
I didn't see any changes to tests\sample\sample.vcxproj

Reply:
Well played! I did not realize that both a CMake and a "Solution" build existed together. I will take a look! Sorry!!